ui/components/ui/sender-to-recipient/sender-to-recipient.component.js

```diff
@@ -163,6 +164,7 @@ export function RecipientWithAddress({
                 recipientNickname ||
                 recipientMetadataName ||
                 recipientEns ||
+                shortenAddress(checksummedRecipientAddress) ||
```

Comment:
this line ensures that a recipient address is rendered if it exists but does not have an account, addressbook, token or ens name